aten/src/ATen/native/cuda/Embedding.cu

```diff
@@ -17,6 +17,7 @@
 
 #if CUB_SUPPORTS_SCAN_BY_KEY()
 #include <thrust/iterator/reverse_iterator.h>
+#include <thrust/iterator/constant_iterator.h>
```

Comment:
This one is a bit different. Standard C++ defines `max` / `min` as functions. So we moved towards the same naming as thrust had and replaced them with `cuda::{maximum, minimum}`

As these are not standard C++ functors they live in `<cuda/functional>`
```suggestion
#include <cuda/functional>
#include <thrust/iterator/reverse_iterator.h>
#include <thrust/iterator/constant_iterator.h>
```